server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb

```diff
@@ -25,10 +25,10 @@ class MaterialTestController < ::ApiV1::BaseController
         before_action :check_admin_user_or_group_admin_user_and_403
 
         def test
-          material_config = ApiV3::Admin::Pipelines::Materials::MaterialRepresenter.new(ApiV3::Admin::Pipelines::Materials::MaterialRepresenter.get_material_type(params[:type]).new).from_hash(params)
+          material_config = ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.new(ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.get_material_type(params[:type]).new).from_hash(params)
```

Comment:
The material_test_controller was for the check connection button on the quick edit page, right? Why didn't you want this to be ApiV5::Admin::Pipelines::Materials::MaterialRepresenter?

Reply:
You can get rid of the materials directory in api_v4 then

Reply:
It's also used by `ConfigRepoOriginRepresenter`, want me to change that usage too?

Reply:
Yes please.